samples/matter/light_bulb/configuration/nrf52840dk_nrf52840/pm_static.yml

```diff
@@ -1,38 +1,38 @@
 mcuboot:
     address: 0x0
-    size: 0xc000
+    size: 0x6000
     region: flash_primary
 mcuboot_pad:
-    address: 0xc000
+    address: 0x6000
     size: 0x200
 app:
-    address: 0xc200
-    size: 0xebe00
+    address: 0x6200
+    size: 0xf1e00
 mcuboot_primary:
     orig_span: &id001
         - mcuboot_pad
         - app
     span: *id001
-    address: 0xc000
-    size: 0xec000
+    address: 0x6000
+    size: 0xf2000
     region: flash_primary
 mcuboot_primary_app:
     orig_span: &id002
         - app
     span: *id002
     address: 0xc200
-    size: 0xebe00
+    size: 0xf1e00
 settings_storage:
     address: 0xf8000
     size: 0x8000
     region: flash_primary
 mcuboot_secondary:
     address: 0x0
-    size: 0xec000
+    size: 0xf1e00
     device: MX25R64
     region: external_flash
 external_flash:
-    address: 0xec000
-    size: 0x714000
+    address: 0xf1e00
```

Comment:
If I was right regarding mcuboot_secondary size, that one needs to be aligned too.